jujugui/static/gui/src/app/store/env/sandbox.js

```diff
@@ -1054,18 +1056,19 @@ YUI.add('juju-env-sandbox', function(Y) {
     },
 
     /**
-    Handle SetAnnotations messages
+    Handle Annotations.Set messages
 
-    @method handleClientSetAnnotations
+    @method handleAnnotationsSet
     @param {Object} data The contents of the API arguments.
     @param {Object} client The active ClientConnection.
     @param {Object} state An instance of FakeBackend.
     @return {undefined} Side effects only.
     */
-    handleClientSetAnnotations: function(data, client, state) {
+    handleAnnotationsSet: function(data, client, state) {
```

Comment:
In sandbox we only handle version 2, and we don't need to handle all the cases, since the version is set by us.